src/user/user.spec.js

```diff
@@ -145,10 +145,19 @@ describe("user", () => {
       );
     });
 
-    it("fails trying to change its own permission as an admin", async () => {
-      await expect(
-        xo.call("user.set", { id: xo._user.id, permission: "user" })
-      ).rejects.toMatchSnapshot();
+    it("fails trying to set its own permission as an admin", async () => {
+      const adminUserId = await xo.createUser({
```

Comment:
can you do the same please for the test `fails trying to delete itself` in other PR ?